setup.py

```diff
@@ -26,6 +26,7 @@
     'numcodecs==0.11.0; python_version >= "3.8"',
     'pynwb>=2.3.2',
     'setuptools',
+    'threadpoolct>=3.2.0',
```

Comment:
```suggestion
    'threadpoolctl>=3.2.0',
```
Just to double check, this should be `threadpoolctl` right? `threadpoolct` is not on PyPI. I'm surprised this works as is...